### ISSUE TITLE
FN S2857: No issue is raised when in file scoped namespace

### ISSUE THREAD

**Issue by mary-georgiou-sonarsource** [SonarSource/sonar-dotnet#5981]
### Description

When the strings to be checked are in a file scoped namespace, no issues are raised.

### Repro steps

```csharp
using System;
using System.Data.SqlClient;
using Linq = System.Linq;

namespace Tests.Diagnostics;

record Record
{
    public void VariousSqlKeywords()
    {
        var select = "SELECT e.*, f" +
            "FROM DimEmployee AS e" + // FN
            "ORDER BY LastName;"; // FN
    }
}
```

### Related information

* C#/VB.NET Plugins version: 8.43.0.51858
